package.json

```diff
@@ -50,9 +52,23 @@
     "docs:watch": "gitbook install && gitbook serve",
     "prepublish": "nwb build",
     "release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci",
-    "test:ts": "tsfmt -r && tsc -p ./",
-    "test:watch": "nwb test --server"
+    "pretest": "npm run style && npm run test:node && npm run test:ts",
+    "test": "nwb test",
+    "test:watch": "npm test -- --server",
+    "test:coverage": "npm test -- --coverage",
+    "test:ci": "npm test -- --ci",
+    "test:node": "node umd/skatejs.js",
+    "test:ts": "tsc -p ./",
+    "style": "npm run style:ts && npm run style:js",
+    "style:fix": "npm run style:ts:fix && npm run style:js:fix",
+    "style:ts": "tsfmt --verify",
+    "style:ts:fix": "tsfmt -r",
+    "style:js": "semistandard",
+    "style:js:fix": "semistandard --fix",
+    "verify": "npm run style && npm test",
+    "commit": "git-cz",
+    "commitmsg": "conventional-changelog-lint --e",
+    "prepush": "npm run verify"
```

Comment:
I used to use precommit instead of prepush, but it slows your workflow. Every time you commit, you have to wait.

We also discussed it within our team at work and everyone is happy with prepush.

In next PR we can definitely add `lint-staged` to precommit https://github.com/prettier/prettier#pre-commit-hook-for-changed-files